FAQ.md

```diff
@@ -0,0 +1,100 @@
+# Frequently Asked Questions
+
+An index of frequently asked questions regarding all things ECMA-262.
+
+# Process Questions
+
+##### What is the process for proposing a new feature?
+
+New features start life as a proposal to the [TC39](#what-is-a-tc39) committee and must be championed (or co-championed) by at least one member of the committee. Once the proposal is raised at a committee meeting, it will become a Stage 0 proposal and move along from there. For more details on how proposal stages work, check out the [proposal process document][proposal-process-document].
+
+If you would like to contribute, please check out [Contributing to ECMAScript](https://github.com/jeffmo/ecma262/blob/master/CONTRIBUTING.md).
+
+##### What is a "TC39"?
+
+TC39 stands for "Technical Committee 39" and is the committee responsible for iterating on and evolving the ECMAScript language specification. The committee generally meets around 6 times a year to discuss progress on pending proposals and collectively work on moving forward with changes to the spec.
+
+##### Why can't we remove feature X?
+
+Changes to ECMAScript must carefully consider the state of the world using the previous version of the language. This includes a large percentage of the web. As a result, in order to remove a feature from ECMAScript, TC39 must be able to show that the feature is used almost never (and thus can be removed). Going through this exercise is extremely difficult and sometimes impossible -- so in general ECMAScript *very* rarely removes features.
+
+# Feature Questions
+
+### Arrow Functions
+
+##### Why isn't there a `->` version of arrow functions?
+
+The motivation for `=>` was to address the oft-fired footgun of dynamic `this` bindings. Additionally, having two forms of arrows is confusing; So only one form was added.
+
+### Destructuring
+
+##### Why isn't the object property destructuring syntax flipped the other way?
+
+(i.e. `let {x: y} = {x: 42}` vs `let {y: x} = {x: 42}`)
+
+In all other object patterns in the language, the syntax to the left of the colon represents the "structure" of an object; So having destructuring patterns match this convention was most consistent.
+
+More fundamentally, however, flipping the syntax the other way would produce a grammar that requires infinite lookahead to properly disambiguate.
```

Comment:
Yea, I suppose. But I was a bit thrown off by the "More fundamentally" in the second paragraph, which never even was a relevant consideration in this case.